
map_anon_fun.erl:4: Function g/1 will never be called
